pkg/notifier/desktop_windows.go

```diff
@@ -6,6 +6,8 @@ import (
 	"github.com/go-toast/toast"
 )
 
+const NameDesktopNotifier = "desktop"
```

Comment:
Does it have to be duplicated? It's already defined in `desktop.go`
btw. I've told you I'll CR it like.. now :D